content/applications/inventory_and_mrp/manufacturing/management/product_variants.rst

```diff
@@ -1,23 +1,35 @@
-=======================================
-How to manage BoMs for product variants
-=======================================
+=======================
+Variants BOM Management
+=======================
 
 Odoo allows you to use one bill of materials for multiple variants of
-the same product. Simply enable variants from
-:menuselection:`Configuration --> Settings`.
+the same product. To activate variants, simply go to :menuselection:
+`Configuration --> Settings --> Variants`.
```

Comment:
Spaces between the `menuselection` directive and the selection break the directive.
```suggestion
the same product. To activate variants, simply go to :menuselection:`Configuration --> Settings
--> Variants`.
```